README.md

```diff
@@ -20,12 +20,14 @@ This document assumes familiarity with the XMOS xCORE architecture, the XMOS too
 
 ## Building
 For an XCore build:
+
     mkdir -p build_xcore
     cd build_xcore
     cmake -DCMAKE_TOOLCHAIN_FILE=../etc/xmos_toolchain.cmake ..
     make
```

Comment:
can these changes be removed?

Reply:
The newline makes it render as a code block instead of a paragraph